docs/store-operations/catalog/graphql-admin/store-metafields

```diff
@@ -0,0 +1,134 @@
+# Store metafields
+
+Store metafields allow you to attach a metafield to the store itself. They are suitable for any global information against the store that is not channel specific. Channel metafields are appropriate for any storefront-specific data you need to store in metafields. There are two ways to create a store metafield in Admin GraphQL. You can remove the resource ID or use the store’s Global ID. Examples of each type are below:
```

Comment:
@zvuki  do you think we should recommend one way of doing this over another?

Reply:
@zvuki what is the preferred way?